### ISSUE TITLE
always get zero for observation.ego_vehicle_state.yaw_rate

### ISSUE THREAD

**Issue by YuechengLiu** [huawei-noah/SMARTS#1481]
While trying to calculate actions for expert data, I find that `observation.ego_vehicle_state.yaw_rate` always return `0.0`. 

This is my agent interface, and I think the required sensors have been opened.
```python
interface = AgentInterface(
    done_criteria=done_criteria,
    max_episode_steps=None,
    waypoints=True,
    neighborhood_vehicles=True,
    ogm=False,
    rgb=False,
    lidar=False,
    action=ActionSpaceType.Imitation,
)
```

**Comment by Gamenot** [comment]
Hello, @YuechengLiu this would appears be a bug. Could you list the version of smarts where this occurs?

**Comment by Gamenot** [comment]
I have assigned @sah-huawei to this issue.

**Comment by YuechengLiu** [comment]
Thanks. What I am using is the latest `develop` branch. But I find that the problem exists even for a very old SMARTS version.

**Comment by sah-huawei** [comment]
I've found the problem and fixed it on the `smarts_traffic` branch, which is slated to go into v0.7.  I think we can also make the fix independently for our next v0.6.1 or v0.6.2 release if necessary.  @Gamenot :  what is our release schedule for these?

**Comment by YuechengLiu** [comment]
Thanks! Maybe you could merge the change into the `develop` branch first.

**Comment by turix2** [comment]
> Thanks! Maybe you could merge the change into the `develop` branch first.

Yes, we will do that before its included in a release.  (We normally merge first into `develop`, then upon release, into `master`.)

However, the `smarts_traffic` branch cannot yet be merged into `develop` since it is a "breaking change" requiring a full version bump (to version `0.7` from version `0.6`) instead of an incremental change (which would go from, for example, `0.6.1` to `0.6.2`).  Once `smarts_traffic` is merged into `develop`, the next release must be `0.7`.

The reason I did this fix on the `smart_traffic` branch first is because it is somewhat easier and cleaner there, since this branch includes a major refactor to how we handle Providers, and this particular bug relates to that.  But the fix can also be made in a separate PR that gets merged into `develop` sooner, although it may not be as clean there.  I'm willing to do so if it seems like version `0.7` will be delayed for a while.

**Comment by Gamenot** [comment]
We could release 6.1, tomorrow I believe. That would let us unblock merging.

**Comment by sah-huawei** [comment]
The fix for this has now been merged into `develop`.
